setup.py

```diff
@@ -0,0 +1,66 @@
+from setuptools import setup, find_packages
+from codecs import open
```

Comment:
This isn't needed in Python 3 I don't think. You can just use the `open` builtin, as the default character encoding in Python 3 is UTF-8.